app/src/test/java/pack/cpserver/db/DbBackendTest.java

```diff
@@ -22,7 +22,8 @@
 import solid.collectors.ToSolidList;
 import solid.stream.Stream;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 @RunWith(RobolectricTestRunner.class)
 @Config(constants = BuildConfig.class)
```

Comment:
Между запусками бд? Не знал об этом, первый раз использую роболектрик.

чт, 11 авг. 2016, 17:59 matreshkin <notifications@github.com>:

> In app/src/test/java/pack/cpserver/db/DbBackendTest.java
> <https://github.com/artemohanjanyan/CPServer/pull/1#discussion_r74438455>:
>
> > @@ -22,7 +22,8 @@
> >  import solid.collectors.ToSolidList;
> >  import solid.stream.Stream;
> >
> > -import static org.junit.Assert.*;
> > +import static org.junit.Assert.assertEquals;
> > +import static org.junit.Assert.assertTrue;
> >
> >  @RunWith(RobolectricTestRunner.class)
> >  @Config(constants = BuildConfig.class)
>
> Выглядит странно. Роболектрик должен сам следить за тем, чтобы БД была
> чистая, на него можно положиться.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/artemohanjanyan/CPServer/pull/1/files/44bb865faaf073fb1a2a813f1734a1f2d28fc658#r74438455>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHH7Y6T6L-3-aKSajUEac8ql3r4zVN40ks5qezjTgaJpZM4Jh_RJ>
> .
>